### PR TITLE
fix: small usability issues and styles

### PR DESCRIPTION
**Pull request by cabreraalex**

### CODE REVIEW

File: src/lib/components/Instances/InstanceTableRow.svelte [hunk 1]
```diff
@@ -33,6 +33,12 @@
 			metric = getMetric(prompt, localInstanceLabel || '', pred || '', project.metric_name);
 		});
 	});
+
+	$effect(() => {
+		if (localInstanceInput.length === 0) {
+			inputArea?.focus();
```
Comment (cabreraalex) [inline src/lib/components/Instances/InstanceTableRow.svelte:39]: focuses input when first added to highlight you should write something

File: src/lib/components/prompt/PromptSuggestion.svelte [hunk 2]
```diff
@@ -80,10 +80,10 @@
 			{/if}
 		{/if}
 	</div>
-	<div class="mt-2 flex items-center justify-end gap-4">
+	<div class="mt-2 flex items-center justify-end gap-3">
 		{#if !applied && !applyingSuggestion}
 			<button
-				class="cursor-pointer text-gray-inactive transition {!disabled
+				class="cursor-pointer text-sm text-gray-active transition {!disabled
```
Comment (cabreraalex) [inline src/lib/components/prompt/PromptSuggestion.svelte:86]: we disagree on this but I think the disabled color is too light for this. I made the text smaller instead

Comment (Sparkier) [reply src/lib/components/prompt/PromptSuggestion.svelte:86]: lgtm like this

File: src/lib/components/prompt/PromptSuggestion.svelte [hunk 3]
```diff
@@ -97,7 +97,7 @@
 		{:else if applyingSuggestion}
 			<Spinner />
 		{:else}
-			<Button {disabled} onclick={accept}>Apply</Button>
+			<Button classNames="h-8" {disabled} onclick={accept}>Apply</Button>
```
Comment (Sparkier) [inline src/lib/components/prompt/PromptSuggestion.svelte:100]: why does this need a height? Shouldn't the items-center center this?

**Review by Sparkier — APPROVED** [review]

File: src/lib/components/prompt/PromptSuggestion.svelte (hunk 3 above)
Comment (cabreraalex) [reply src/lib/components/prompt/PromptSuggestion.svelte:100]: just to make it slightly shorter than other buttons I thought it looked a bit nicer.

Comment (Sparkier) [reply src/lib/components/prompt/PromptSuggestion.svelte:100]: ah ooops I thought this was dismiss, never mind

**Pull request merged**